src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/VoteOutcome/VoteResults/VoterAvatars.tsx

```diff
@@ -22,16 +23,25 @@ const displayName =
 
 const VoterAvatars = ({ voters, maxAvatars }: VoterAvatarsProps) => {
   const remainingAvatarsCount = calculateRemainingItems(maxAvatars, voters);
+  const { user } = useAppContext();
   const voterAddresses = useMemo(
     // We need a stable reference to this array to avoid an infinite loop in `useGetUsers`
     () =>
       voters
-        .map(({ address }) => address)
+        .reduce((acc, { address }) => {
```

Comment:
You could use the generic `reduce` takes:
```suggestion
        .reduce<string[]>((acc, { address }) => {
```
instead of casting below